kubectl-minio/static/crd.yaml

```diff
@@ -0,0 +1,161 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This CRD yaml is outdated with the kustomize version @nitisht

Reply:
Updated the CRD, but we should keep it to v1 instead of v1beta1

Reply:
The reason we keep it v1beta1 is to cater for 1.16 or 1.17 deployments which are quite a lot and of course NOTE: the crd is autogenerated so there is a reason why they kept it v1beta1

Reply:
So we should avoid this double copy of crd's in the project and keep it in one place follow the autogenerated code to do this.

Reply:
> The reason we keep it v1beta1 is to cater for 1.16 or 1.17 deployments which are quite a lot and of course '

The client-go library that we use is 1.18 so we are only promising compatibility with 1.17 anyways. Also `apiextensions.k8s.io/v1` needs k8s 1.16 and above, so we are already covered. Ref:
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

Reply:
> NOTE: the crd is autogenerated so there is a reason why they kept it v1beta1

This may be incorrect setup, how is it autogenerated?

Reply:
@nitisht I don't think it's incorrect

```
regen-crd:
        @controller-gen crd:trivialVersions=true paths="./..." output:crd:artifacts:config=operator-kustomize/crds/
```

Reply:
https://github.com/kubernetes-sigs/controller-tools install from here

Reply:
understood. With removal of kustomize file, we will point to a single CRD for the operator and plugin. So there will be no duplication of file.